src/index.d.ts

```diff
@@ -982,6 +987,123 @@ declare namespace admin.remoteConfig {
    */
   type RemoteConfigParameterValue = ExplicitParameterValue | InAppDefaultValue;
 
+  /**
+   * Interface representing a Remote Config template version.
+   * Output only, except for the version description. Contains metadata about a particular
+   * version of the Remote Config template. All fields are set at the time the specified Remote
+   * Config template was published. A version's description field may be specified in
+   * `publishTemplate` calls.
+   */
+  export interface Version {
+    /**
+     * The version number of a Remote Config template.
+     */
+    versionNumber?: string;
+
+    /**
+     * The timestamp of when this version of the Remote Config template was written to the
+     * Remote Config server.
+     */
+    updateTime?: string;
+
+    /**
+     * The source of origin of the template update action.
+     */
+    updateOrigin?: ('REMOTE_CONFIG_UPDATE_ORIGIN_UNSPECIFIED' | 'CONSOLE' |
+      'REST_API' | 'ADMIN_SDK_NODE');
+
+    /**
+     * The type of the template update action.
+     */
+    updateType?: ('REMOTE_CONFIG_UPDATE_TYPE_UNSPECIFIED' |
+      'INCREMENTAL_UPDATE' | 'FORCED_UPDATE' | 'ROLLBACK');
+
+    /**
+     * Aggregation of all metadata fields about the account that performed the update.
+     */
+    updateUser?: RemoteConfigUser;
+
+    /**
+     * The user-provided description of the corresponding Remote Config template.
+     */
+    description?: string;
+
+    /**
```

Comment:
Suggest a bit of a rewrite:

"The version number of
the Remote Config template that has become the current version due to a  rollback. Only present if this version is the result of a rollback."

Is this accurate?  :)

Reply:
This is accurate! Thanks. Updated.